src/SimpleCalculator/Addition.cs

```diff
@@ -2,10 +2,11 @@ namespace SimpleCalculator;
 
 public class Addition
 {
-    public static int Add()
+    UserInputs inputs = new UserInputs();
+    public int Add()
```

Comment:
basically method `Add` is responsible for calculation only, so getting input numbers is not appropriate there. Please pass variables as parameters